cdc_rsync/cdc_rsync_client.cc

```diff
@@ -109,24 +109,26 @@ GgpRsyncClient::GgpRsyncClient(const Options& options, PathFilter path_filter,
                     kForwardPortFirst, kForwardPortLast, &process_factory_,
                     &remote_util_),
       printer_(options.quiet, Util::IsTTY() && !options.json),
-      progress_(&printer_, options.verbosity, options.json) {}
+      progress_(&printer_, options.verbosity, options.json) {
+  if (options_.ssh_command) {
+    remote_util_.SetSshCommand(options_.ssh_command);
+  }
+  if (options_.scp_command) {
+    remote_util_.SetScpCommand(options_.scp_command);
+  }
+}
 
 GgpRsyncClient::~GgpRsyncClient() {
   message_pump_.StopMessagePump();
   socket_.Disconnect();
 }
 
 absl::Status GgpRsyncClient::Run() {
-  absl::Status status = remote_util_.GetInitStatus();
-  if (!status.ok()) {
-    return WrapStatus(status, "Failed to initialize critical components");
-  }
-
   // Initialize |remote_util_|.
-  remote_util_.SetIpAndPort(options_.ip, options_.port);
+  remote_util_.SetHostAndPort(options_.ip, options_.port);
```

Comment:
Can you change `Options::ip` to `Options::host`?

Reply:
I'm calling this user_host in a later CL in this chain.